### ISSUE TITLE
NSLogger in "Remote" (Bonjour) Mode - Logger Appears At Different Times

### ISSUE THREAD

**Issue by bleeckerj** [fpillet/NSLogger#82]
So — I've been doing "remote" logging with a device on the same network as my debug station. I've noticed that the Logger seems to connect (open a window, indicate that it has an incoming connection) seemingly at random times, almost always well-after the Logger in my App has been loaded and has started logging. It seems to always appear at some time — but often quite a-ways after I've started.

I can continue to figure out what's what, but I wonder — has anyone else experienced this? Is there a specific way to get the default logger to get up on its feet? Currently I'm doing a simple incantation:

```
LoggerSetOptions(NULL, kLoggerOption_BrowseBonjour|kLoggerOption_UseSSL);
LoggerStart(LoggerGetDefaultLogger());
```

and then I immediately log something through a macro.

It all works quite normally and as expected when I'm logging with the iOS Simulator, for example.

Any suggestions?

Julian

**Comment by haemi** [comment]
any news on this? Does this work now?

**Comment by fpillet** [comment]
@haemi LoggerSetBufferFile logs to a file, not to the network so it's a totally different thing. I know a few users who do file logging, and it works for them. Can you open a new issue and show some code you're using to setup logging ?

Note that it's best to set the buffer file before starting to log ... but it should work nonetheless.
